src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,68 @@
-export const Tabs = () => {};
+import { useEffect } from 'react';
+import { Tab } from '../../types/tab';
+
+type Props = {
+  tabs: Array<Tab>,
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs = ({
+  tabs, selectedTabId, onTabSelected,
+}: Props) => {
+  useEffect(() => {
+    if (!tabs.map(el => el.id).includes(selectedTabId)) {
+      document.querySelector('li:first-child')?.classList.add('is-active');
+    }
+  });
```

Comment:
try to use `useEffect` only when really necessary, I think it could be done with simple `handleTabClick` function